docs/src/content/docs/fr/reference/configuration.mdx

```diff
@@ -444,6 +444,9 @@ starlight({
 });
 ```
 
+Les entrées dans `head` sont converties directement en éléments HTML et ne passent pas par le traitement [de regroupement de script](https://docs.astro.build/fr/guides/client-side-scripts/#regroupement-de-script) ou le [style](https://docs.astro.build/fr/guides/styling/#styliser-avec-astro) d'Astro.
+Si vous avez besoin d'importer des ressources locales comme des scripts, des styles ou des images, [réutilisez le composant Head](/fr/guides/overriding-components/#réutiliser-un-composant-intégré).
```

Comment:
```suggestion
Si vous avez besoin d'importer des ressources locales comme des scripts, des styles ou des images, [redéfinissez le composant Head](/fr/guides/overriding-components/#réutiliser-un-composant-intégré).
```

In the English version, even if we link to the "reuse …" anchor, "override" is still the main action to emphasize.